replace_words.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/python
+import sys
+import itertools
+import os
+import argparse
+
+
+class WordsModifier:
+    default_input_file = 'file1.txt'
+
+    def __init__(self, input_file='file1.txt', output_file='file2.txt',
+                 from_list_arr='no,bad', to_list_arr='yes,happy'):
+        self.__input_file = input_file
+        self.__output_file = output_file
+        self.__from_list_arr = from_list_arr
+        self.__to_list_arr = to_list_arr
+
+    def get_input_file(self):
```

Comment:
it's preferred that you use properties rather than set/get method definitions. 
@property
@variable.setter